cordova/cordova-feature-android-tests/feature/comm.py

```diff
@@ -57,6 +57,13 @@ def setUp():
         ARCH = "x86"
     f_arch.close()
 
+    f_mode = open(const_path + "/../mode.txt", 'r')
+    if f_mode.read().strip("\n\t") != "shared":
```

Comment:
Better to make the judgement accurate to avoid the situation that the mode is in wrong spelling.